src/ngraph/op/op.tbl

```diff
@@ -0,0 +1,112 @@
+//*****************************************************************************
```

Comment:
I think `op_tbl.hpp` would be better for editors that use file extension for syntax selection.

Reply:
done